.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Themes/Simply-Love"]
+	path = Themes/Simply-Love
```

Comment:
I think the path should just be `Simply Love` (with a space instead of a dash) since that's how our releases are

Reply:
moved!